cmd/geth/main.go

```diff
@@ -261,12 +262,28 @@ func prepare(ctx *cli.Context) {
 	// If we're a full node on mainnet without --cache specified, bump default cache allowance
 	if ctx.GlobalString(utils.SyncModeFlag.Name) != "light" && !ctx.GlobalIsSet(utils.CacheFlag.Name) && !ctx.GlobalIsSet(utils.NetworkIdFlag.Name) {
 		// Make sure we're not on any supported preconfigured testnet either
-		if !ctx.GlobalIsSet(utils.TestnetFlag.Name) && !ctx.GlobalIsSet(utils.RinkebyFlag.Name) && !ctx.GlobalIsSet(utils.GoerliFlag.Name) && !ctx.GlobalIsSet(utils.DeveloperFlag.Name) {
+		if !ctx.GlobalIsSet(utils.LegacyTestnetFlag.Name) && !ctx.GlobalIsSet(utils.RopstenFlag.Name) && !ctx.GlobalIsSet(utils.RinkebyFlag.Name) && !ctx.GlobalIsSet(utils.GoerliFlag.Name) && !ctx.GlobalIsSet(utils.DeveloperFlag.Name) {
 			// Nope, we're really on mainnet. Bump that cache up!
 			log.Info("Bumping default cache on mainnet", "provided", ctx.GlobalInt(utils.CacheFlag.Name), "updated", 4096)
 			ctx.GlobalSet(utils.CacheFlag.Name, strconv.Itoa(4096))
 		}
 	}
+	// If we're running a known preset, log it for convenience.
+	if ctx.GlobalIsSet(utils.LegacyTestnetFlag.Name) {
+		log.Info("Starting Geth on Ropsten testnet...")
+		log.Warn("The --testnet flag is ambiguous! Please specify one of --goerli, --rinkeby, or --ropsten.")
+		log.Warn("The generic --testnet flag is deprecated and will be removed in the future!")
+	} else if ctx.GlobalIsSet(utils.RopstenFlag.Name) {
+		log.Info("Starting Geth on Ropsten testnet...")
+	} else if ctx.GlobalIsSet(utils.RinkebyFlag.Name) {
+		log.Info("Starting Geth on Rinkeby testnet...")
+	} else if ctx.GlobalIsSet(utils.GoerliFlag.Name) {
+		log.Info("Starting Geth on Görli testnet...")
+	} else if ctx.GlobalIsSet(utils.DeveloperFlag.Name) {
+		log.Info("Starting Geth in ephemeral dev mode...")
+	} else if !ctx.GlobalIsSet(utils.NetworkIdFlag.Name) {
+		log.Info("Starting Geth on Ethereum mainnet...")
+	}
```

Comment:
When doing more than 1 `else` branch, please use a `switch` statement:

```go
switch {
case ctx.GlobalIsSet(utils.LegacyTestnetFlag.Name):
	log.Info("Starting Geth on Ropsten testnet...")
	log.Warn("The --testnet flag is ambiguous! Please specify one of --goerli, --rinkeby, or --ropsten.")
	log.Warn("The generic --testnet flag is deprecated and will be removed in the future!")

case  ctx.GlobalIsSet(utils.RopstenFlag.Name):
	log.Info("Starting Geth on Ropsten testnet...")

case  ctx.GlobalIsSet(utils.RinkebyFlag.Name):
	log.Info("Starting Geth on Rinkeby testnet...")

case  ctx.GlobalIsSet(utils.GoerliFlag.Name):
	log.Info("Starting Geth on Görli testnet...")

case  ctx.GlobalIsSet(utils.DeveloperFlag.Name):
	log.Info("Starting Geth in ephemeral dev mode...")

case  !ctx.GlobalIsSet(utils.NetworkIdFlag.Name):
	log.Info("Starting Geth on Ethereum mainnet...")
}
```

Also since there is no "natural" separator between the lines here (i.e. no `{}` inside the cases), you might want to leave an empty line between the case statements. Makes reading the code easier on the eye.